.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "env": {
+      "browser": true,
+      "es6": true,
+      "jest": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "extends": ["airbnb-base"],
+    "rules": {
+      "no-shadow": "off",
+      "no-param-reassign": "off",
+      "eol-last": "off",
+      "arrow-parens": "off",
+      "func-names": "off"
```

Comment:
- [ ] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team. Check it [out](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png)

Let's remove `func-names` rule.

Reply:
Done.